### DISCUSSION TITLE
Support für andere Server

### DISCUSSION THREAD

**Discussion by PianoRailways** [topological-state/stskit#62] [Ideas]
Bei jTraingraph ist es z.B. möglich, zu definieren, auf welchen Server (welche IP im Netzwerk) das Plugin hören soll.

**Comment by topological-state** [comment]
Mit der Kommandozeilenoption --host ist das bereits möglich, s. https://github.com/topological-state/stskit/wiki/Installation#kommandozeilen-optionen
